src/colours.scss

```diff
@@ -92,3 +92,6 @@ $rpf-darkmode-text-secondary: #CCCCCC;
 // New Colours
 $rpf-grey-150: #D5D7DC;
 $rpf-grey-600: #4A4D59;
+
+$rpf-text-secondary: #4D575B;
+$rpf-text-secondary-dark: #CCCCCC;
```

Comment:
just check in with @magdalenajadach and see if these need redefining in her PR 👍